swagger.yaml

```diff
@@ -5558,6 +5565,7 @@ definitions:
     type: string
     enum:
       - 'connected'
+      - 'media-connected'
```

Comment:
@jkilzi Do we need to UI code when we add new validation?

Reply:
No, but we'll need to update our types. @celdrake can you PTAL?

Reply:
I'll update them in `assisted-ui-lib`